calibrate/manual.py

```diff
@@ -28,6 +32,9 @@ def ros_setup(self):
         self.pub_reset   = rospy.Publisher(topic + '/reset_heading', Int16, queue_size=1)
         self.pub_set_pos = rospy.Publisher(topic + '/set_position', Point, queue_size=1)
 
+        self.sub_center = rospy.Subscriber('/arena' + topic + '/center_mm', PointStamped, self.set_tracker_estimate,
```

Comment:
Disadvantage is now the tracker has to be up to make this usable; I wonder if there's a way to say try to subscribe but ignore if not available.  Not suggesting you change it, for most persons this is fine.  

My original intent with 'manual' was just to be able to drive the sphero around and understand what it was doing.